src/modules/billing/services/transactions-service.js

```diff
@@ -79,19 +77,56 @@ const setErrorStatus = transactionId =>
     status: Transaction.statuses.error
   });
 
-const updateTransactionVolumes = async batch => {
-  const transactions = await newRepos.billingTransactions.findByBatchId(batch.id);
-  const billingVolumes = await billingVolumesService.getVolumesForBatch(batch);
-  for (const billingVolume of billingVolumes) {
-    const transaction = transactions.find(trans =>
-      trans.chargeElementId === billingVolume.chargeElementId);
-    if (!transaction) throw new NotFoundError(`No transaction found for billing volume ${billingVolume.billingVolumeId}`);
-    await newRepos.billingTransactions.update(transaction.billingTransactionId, { volume: billingVolume.volume });
-  }
+const updateDeMinimis = (ids, isDeMinimis) =>
+  newRepos.billingTransactions.update(ids, { isDeMinimis }, false);
+
+/**
+ * Clears the de-minimis flags by transaction IDs
+ * @param {Array<String>} externalIds
+ * @return {Promise}
+ */
+const clearDeMinimisByTransactionIds = partialRight(updateDeMinimis, false);
+
+/**
+ * Sets the de-minimis flags by transaction IDs
+ * @param {Array<String>} externalIds
+ * @return {Promise}
+ */
+const setDeMinimisByTransactionIds = partialRight(updateDeMinimis, true);
```

Comment:
I don't think these functions are used anywhere. Did you mean to use them below?

Reply:
Thanks, this should be fixed now